spec/spec_helper.rb

```diff
@@ -72,7 +72,8 @@
   #   # is tagged with `:focus`, all examples get run. RSpec also provides
   #   # aliases for `it`, `describe`, and `context` that include `:focus`
   #   # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
-  #   config.filter_run_when_matching :focus
+  config.filter_run_when_matching :focus
+  config.filter_run_excluding vpn_only: true if ENV['VPN'].nil?
```

Comment:
this auto filters the vpn only spec based on env. no more forgetting to set the filter =-)

Reply:
Nice :)